resources/views/components/card/address.blade.php

```diff
@@ -2,18 +2,19 @@
 
 <address-card
     {{ $attributes->whereStartsWith('v-') }}
-    @if(!$dynamicType)
-        :shipping="{{ var_export($shipping) }}"
-        :billing="{{ var_export($billing) }}"
-    @endif
+    @if (!$dynamicType) :shipping="{{ var_export($shipping) }}"
+        :billing="{{ var_export($billing) }}" @endif
     v-slot="{ address, billing, shipping, isEmpty }"
 >
-    <div {{ $attributes->whereDoesntStartWith('v-')->class('relative') }} v-if="!isEmpty">
-        <div class="h-full flex flex-col">
-            @if($check)
+    <div
+        {{ $attributes->whereDoesntStartWith('v-')->class('relative') }}
+        v-if="!isEmpty"
+    >
+        <div class="flex h-full flex-col">
```

Comment:
i don't like h-full, can you re check if you can fix this with flex-1 because of browser version compatibility